File: JavaScript_Courses/JavaScript_Basics/JS_BP_2022_10/01_First_Steps_In_Coding_Exercise/02_Radians_to_Degrees.js
function radiansToDegrees(input) {  // input = ["6.2832"] ; array с 1 елемент, к е на индекс 0 и е "6.2832"
    let radians = Number(input[0]);  // input[0] = "6.2832"; Number(input[0]) == Number("6.2832") == 6.2832
    let degrees = radians * 180 / Math.PI;

    console.log(degrees);
}

// radiansToDegrees(["6.2832"]);

File: Python_Courses/Python_Basics/Python_PB_2025_01/01_First_Steps_in_Coding_Exercise/02_Radians_to_Degrees.py
# градус = радиан * 180 / π
# Числото π в Python може да достъпите чрез модула math
# import math  # math.pi
from math import pi


# 1. прочитаме вход; + го кастваме към подходящия дата тип
radians = float(input())
# 2. правим математическото преобразувание
degrees = radians * 180 / pi
# 3. извеждаме на изхода
print(degrees)
